package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-revolve",
-  "version": "3.0.0",
+  "version": "4.0.0-beta.1",
```

Comment:
I do not feel great about pushing `beta` version to the `master` branch

Reply:
ok i will change it to 4.0.0 then